api/src/main/java/ca/bc/gov/educ/api/pendemog/migration/service/StudentService.java

```diff
@@ -80,18 +77,19 @@ private void processChunk(final List<PenDemographicsEntity> penDemographicsEntit
         } else {
           mappedStudentRecord = studentMapper.toStudent(penDemog);
         }
-
         try {
           final LocalDate dob = LocalDate.parse(penDemog.getStudBirth());
           mappedStudentRecord.setDob(dob);
         } catch (final Exception e) {
+          log.error("Data Quality Issue, Setting Birthdate of Student to 2000-01-01 for pen :: {}", mappedStudentRecord.getPen());
```

Comment:
In all cases where you're in the catch block, please print out the value which is not able to be parsed with the ID context like you did with the PEN here.